src/pages/index.js

```diff
@@ -162,40 +163,44 @@ const createCard = (data) => {
       isLiked: data.isLiked,
       name: data.name,
       link: data.link,
-      handleImageClick: (imgData) => {
-        cardPreviewModal.open(imgData);
+      handleImageClick: (name, link) => {
+        cardPreviewModal.open(name, link);
+      
       },
 
       handleRemoveCardClick: () => {
         const id = cardEl.getId();
-        newRemoveCardModal.open();
-
-        newRemoveCardModal.setSubmitAction(() => {
-          newRemoveCardModal.setLoadingText(true);
-          api.deleteCard(id).then((res) => {
-            cardEl.handleRemoveCard(res);
-
-          }) .catch((err) => {
-            console.error(err);
-          })
-          .finally(() => {
-            newRemoveCardModal.setLoadingText(false);
-          });
+        removeCardModal.open();
+
+        removeCardModal.setSubmitAction(() => {
+          removeCardModal.renderLoading(true);
+          api
+            .deleteCard(id)
+            .then((res) => {
+              cardEl.handleRemoveCard(res);
+              removeCardModal.close();
+            })
+            .catch((err) => {
+              console.error(err);
+            })
+            .finally(() => {
+              removeCardModal.renderLoading(false);
+            });
         });
       },
       handleLikeIcon: (card) => {
         if (!card.getLikeStatus()) {
-          return api
+           api
             .addLike(card._id)
             .then(() => {
-              card.getLikeStatus(true);
+              card.setLikeStatus(true);
             })
             .catch(console.err);
```

Comment:
`.catch(console.err);` doesn't work. You need to use `.catch(console.error);` for handling errors

**Please, correct it everywhere**